### ISSUE TITLE
Scanning status text shows titles of unshared content

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#4855]
If I am on the web settings while a folder scan is running on a directory I do not have access to, I can see the names of the content

**Comment by SurfaceS** [comment]
`MediaScanner` has access to every active shared.
So, even if the logged account is not allowed to play the content, it will see the current scanned folder.

BTW, if the logged user has access to the settings, he also has access to the log (where shared content, played, scan etc is advised), shared settings (where shared content are visible), etc.

**Comment by SubJunk** [comment]
Yeah I think we should hide the scanning text from users who don't have access to it

**Comment by SubJunk** [comment]
This is unnecessary right now because we disabled the scanning text on the web interface due to a Mantine bug

If that ever gets re-added this would become relevant again, but the status text for scanning isn't a very important feature IMO

**Comment by Mik-S-UMS** [comment]
Ah that explains why it was not showing for me the other day.
Maybe instead of showing the text of what it is actually being scanned, just have some indicator in the UI that it is actually scanning. Something like a spinning progress icon saying "File scan in progress" and it will not need to be updated till it is finished.

**Comment by SubJunk** [comment]
Yeah that's a good idea too, can you make an issue for it?
